src/compas_blender/utilities/misc.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
can be removed. only necessary for Rhino because of Python 2.7...

Reply:
oh, of course.  old habits die hard